app/views/hospitals/_index.html.haml

```diff
@@ -6,7 +6,11 @@
       .list-name= hospital.name
       .list-button-area
         %button.button Remove
-  .list-grid
-    %input.form-input-field{placeholder: "Add Hospitals"}
-    .list-button-area
-      %button.button Add
\ No newline at end of file
+  = form_with url: hospitals_path, scope: :hospital  do |f|
+    .list-grid
+      .list-name
+      = f.text_field :name, id: "Name", class: "form-input-field"
```

Comment:
If `.list-grid` is supposed to be a wrapper around the `link_to`, this line needs to be intended

Reply:
I cannot see a ``` link_to ``` in this file, I think the file changed when we merged the fix PR